dapr/clients/__init__.py

```diff
@@ -173,3 +175,27 @@ async def invoke_method_async(
         else:
             raise NotImplementedError(
                 'Please use `dapr.aio.clients.DaprClient` for async invocation')
+
+    def wait(self, timeout_s: float):
```

Comment:
This is float because DaprGrpcClient already has a method wait as float, I can remove that method if that would be better....